packages/apps/shopify-api/docs/guides/session-storage.md

```diff
@@ -294,4 +292,14 @@ const session = Session.fromPropertyArray(sessionProperties);
 >
 > The existing [SQL-based implementations](../../../session-storage#readme), i.e., MySQL, PostgreSQL and SQLite, convert it from seconds from storage. The remaining implementations do not change the retrieved `expires` property.
 
+### Encrypting data for storage
+
+If you want to encrypt your access tokens before storing them, the `Session` class provides two methods: `fromEncryptedPropertyArray` and `toEncryptedPropertyArray`.
+
+These behave the same as their non-encrypted counterparts, and take in a `CryptoKey` object. If a session is currently not encrypted in storage, these methods will still load it normally, and will encrypt them when saving them.
+Currently, only the `accessToken` is encrypted.
```

Comment:
Question: Should we be making it flexible to allow folks to encrypt more than just the accessToken?

Reply:
I thought of that too, but haven't thought of _how_ the app would convey that information. I think it would need to be a direct argument to `to`/`fromEncryptedArray` that the app would have to pass in to the session storage.

That would mean the app could do something like:

```ts
// Evergreen way of encrypting every column we allow
new PrismaSessionStorage(prisma, {encryptionKey: true, encryptColumns: 'all'});

// Explicitly select which columns
new PrismaSessionStorage(prisma, {encryptionKey: true, encryptColumns: ['accessToken', 'userId']});
```

Reply:
It is fairly easy to implement though, I'm going to timebox it and see if I can get it off the ground.